test/test_quantization.py

```diff
@@ -260,6 +260,166 @@ def checkQuantized(model):
         model = quantize(ManualQuantModel().eval(), test_only_eval_fn, self.calib_data)
         checkQuantized(model)
 
+class PostTrainingDynamicQuantTest(QuantizationTestCase):
+    def test_single_layer_dynamic(self):
+        r"""Dynamic Quantize SingleLayerLinearModel which has one Linear module,
+        make sure it is swapped to nnqd.Linear which is the quantized version of
+        the module
+        """
+        model = SingleLayerLinearModel().eval()
+        qconfig_dict = {
+            '': default_qconfig
+        }
+        model = prepare_dynamic(model, qconfig_dict)
+        # Check if observers are inserted
+        self.checkObservers(model)
+
+        test_only_eval_fn(model, self.calib_data)
```

Comment:
Can we remove this line? We do not need it for dynamic quant.

Reply:
Will update it.

Reply:
Updated.